revolv/static/dashboard.scss

```diff
@@ -399,6 +399,87 @@ $sidebar-icon-gray: #999;
         h6 {
             font-size: 10px;
         }
+
+        .repayment-progress-container {
+            height: 260px;
+            .statistics-graphics-container {
+                height: 260px;
+                width: 100%;
+                position: absolute;
+
+                .svg-graphics-container {
+                // this contains the formatting for the white outside circle - this works
```

Comment:
Random: this should be indented one level. Also the "this-works" part is pretty useless in terms of explanation, so we should probably get rid of it or expand on the explanation more